engine/src/main/java/org/terasology/rendering/dag/nodes/RefractiveReflectiveBlocksNode.java

```diff
@@ -267,4 +286,37 @@ public void update() {
 
         readOnlyGBufferFbo.attachDepthBufferTo(refractiveReflectiveFbo);
     }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent event) {
```

Comment:
I was thinking about this option. Indeed if the method have no arguments the ancestor class would need all sorts of additional local variables. But what if the two handle*() methods above obtained everything they need as input arguments? Feasible?